registrar.py

```diff
@@ -414,28 +352,3 @@ def _required_email_address(cls, uri, problem_title):
             problem.title = problem_title
             return problem
         return uri
-
-    @classmethod
-    def _update_library_authentication_url(
-            cls, library, new_authentication_url,
-            new_opds_url, provided_shared_secret,
-    ):
```

Comment:
This method doesn't work anymore because the authentication_url is modified before the call to register(), and the opds_url is modified afterwards.